src/touchable.tsx

```diff
@@ -128,178 +128,174 @@ export interface PropsMaybeOnPress extends CommonProps {
 }
```

Comment:
better review this ignoring whitespace: https://github.com/Telefonica/mistica-web/pull/551/files?w=1